### ISSUE TITLE
Cannot write large log messages to CloudWatch

### ISSUE THREAD

**Issue by 4z3** [saymedia/journald-cloudwatch-logs#28]
When there are large log messages in the journal, then `journald-cloudwatch-logs` aborts with following error:

```
Failed to write to cloudwatch: failed to put events: InvalidParameterException: Log event too large: 383595 bytes exceeds limit of 262144
         status code: 400, request id: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
```

~~It would be nice if instead large messages get forwarded as multiple chunks.~~ _Edit: Not crashing would be enough already, no need for nice-to-haves :)_

**Comment by whereisaaron** [comment]
This is a [CloudWatch Limit](https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/CalculatePutEventsEntrySize.html) of 256 KB per event. So you have to choose between logging duplicate events for the chunks, or truncating, or (current behavior) not logging at all. I'd suggest calculating the size as per the AWS docs, and truncating if it is too big for a single event.

**Comment by 4z3** [comment]
Yeah, I get that this is an AWS limit. The issue is that journald-cloudwatch-logs aborts/crashes when it fails to send a large log message. It would be already sufficient to not crash (no need for chunking/truncating). I've updated the issue description accordingly.

**Comment by whereisaaron** [comment]
Agreed! Not crashing would be a good improvement 😀
